IPython/html/static/edit/less/edit.less

```diff
@@ -1,3 +1,10 @@
+.header_edit {
+}
+
+.site_edit {
+    height: 88.5%;
```

Comment:
I think I had initially set height: 11.5% in the header but I remembered something was wrong and removed it. Yes, `calc()` should work okay, but a few minutes ago I was trying to get the fixed header using flexbox (doesn't cause the issues in headers with dynamic height). I think it is a very elegant solution and seems to be working but needs more testing. By the way, is there a reason why we have `position: absolute; ` in the body element?